guide/docs/api/tls.mdx

```diff
@@ -42,127 +53,129 @@ these libraries.
   - `rsa_pkcs1_sha512`
   - `rsa_pkcs1_sha384`
   - `rsa_pkcs1_sha256`
-- Client and server name validation through SAN and Common Name.
-- Self-signed certificates (with a special configuration)
+- Client and server name validation through the Common Name or Subject Alternative Name (SAN) extension.
+- Self-signed certificates (with a special configuration).
 
 ## Configuration
 
-TLS configuration is done through the `TlsClientConfig` and `TlsServerConfig` structures. They are both very similar.
-The fields are the following:
-
-- `name`: name requested. The client will advertise this name through
-  a Server Name Indication (SNI) extension in the `Client Hello`. Both
-  the client and the server will verify that the presented certificate
-  is valid for this DNS name. See the following section for more details
-  about name validation. DNS name validation is not performed when
-  `CertificateMode` is `SelfSignedCertificate`.
-- `peer_cert_path`: path to the unencrypted PEM file containing the trusted
-  root certificate(s) or the peer self-signed certificate.
-- `local_cert_path`: path to the unencrypted PEM file containing the certificate
-  or the chain of certificates to present to the peer.
-- `private_key_path`: path to the unencrypted PEM file containing the private
-  key of the local certificate.
-- `password`: password used to decrypt the private key file. Leave empty if the file
-  is not encrypted. See the next section for more details.
-- `min_tls_version`: minimum TLS version to support. Setting this to `Tls1_3`
-  will force TLSv1.3.
-- `certificate_mode`: mode of verification of the presented certificate. See
-  the next section for more details.
-
-### Name validation
-
-Name validation is always performed when `certificate_mode` is `TrustChain`, for
-both the client and the server.
+TLS configuration is configured using the `TlsClientConfig` or `TlsServerConfig` structures. They are very similar and contain the following fields:
+
+- `name`:
+  * The client and the server both verify that the certificate presented by the peer is valid for this name. Check out the next section the gory details on
```

Comment:
```suggestion
  * The client and the server both verify that the certificate presented by the peer is valid for this name. Check out the next section for the gory details on
```
I'm no expert in English, but "gory" sounds scary to me.

Reply:
It's a well-established phrase meaning "the explicit details of something".  Doesn't have a negative connotation in this context.